util-taglib/src/com/liferay/taglib/ui/IconDeleteTag.java

```diff
@@ -91,7 +91,9 @@ protected String getPage() {
 				}
 			}
 
-			setLinkCssClass("component-action");
+			if (!isLabel()) {
+				setLinkCssClass("component-action");
```

Comment:
Looks good to me! 

When taking a look on Clay documentation, I found a very short description for `component-action`:

> Use these patterns for actions in components.

Are "actions" intended to not have text content inside?

Source: https://clayui.com/docs/components/link/markup.html#css-component-action

@matuzalemsteles @pat270 @julien? I can open an issue for a more detailed description :)

Reply:
I think the `component-action` is not necessarily just for text but an button with an icon is also considered an action, @pat270 could explain more.

Reply:
@diegonvs good question: I have no idea of the reply. Maybe @drakonux and has the answer?
I'm going to have a look at the lexicon website to see if I can find more information.